README.md

```diff
@@ -184,13 +184,8 @@ Full info about the Google Drive python API can be found [here](https://develope
   - Click the file_download (Download JSON) button to the right of the client ID.
   - Move this file to your working directory and rename it *"client_secret.json"*
 
-2. Install the Google Client Library
-  - Run the following command to install the library using pip:
-  ```
-  pip install --upgrade google-api-python-client  or pip install --upgrade google-api-python-client-py3

```

Comment:
We do not want them to be installed manually, they are actually installed from `requirements.txt` (with pinned versions).

Reply:
Hello @mjenczmyk, thanks for the input!

The part GOOGLE_DRIVE_API Setup: is about GDrive api setup. Let's keep that info in case of any errors. Sth like:
2. Install the Google Client Library
    packages: ```google-api-python-client``` or ```google-api-python-client-py3``` are required. They shall be already installed via ```pip install -r requirements.txt``` . If not, type the following:
```pip install --upgrade google-api-python-client  or pip install --upgrade google-api-python-client-py3```

What do you think ?

Reply:
I would assume that the user has already run `pip install -r requirements.txt` and because of that we can assume this packages are installed at this point (so there is no need to write it to install them).

If you insist that this point is still needed I would leave it as it is - what should I do.

BTW, I've browsed the net and it seems that `google-api-python-client-py3` was a `google-api-python-client` backport for Python 3 and is no longer supported (`google-api-python-client` seems enough`). On Google Python API website they mention only the second package. Can you confirm that?

Reply:
I would stick to my proposal:
To leave inf like this: 
 2. Install the Google Client Library
package: google-api-python-client  are required. It shall be already installed via pip install -r requirements.txt . If not, type the following:
pip install --upgrade google-api-python-client

Reply:
How about the text below?

```
2. Install the Google Client Library
Package `google-api-python-client` is required. It should be already installed via `pip install -r requirements.txt`. If it's not, type the the command to install it.
```

The point is that we shouldn't want to run `pip install google-api-python-client` as we cannot predict which version will be installed (and it may not work with our script) - the advantage of using requirements file is that dependencies versions are pinned, so we are sure everything will work.